app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_be_correct_visible_convert_to_human_age():
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repetition in your test cases. This will make your tests more concise and easier to maintain .